runtime/near-vm-logic/Cargo.toml

```
@@ -31,7 +31,8 @@ near-vm-errors = { path = "../near-vm-errors", version = "3.0.0" }
 bn = { package = "zeropool-bn", version = "0.5.9", features = [], optional = true }
 
 [dev-dependencies]
-serde_json = {version= "1", features= ["preserve_order"]}
+serde_json = { version = "1", features = ["preserve_order"] }
+serde_with = { version = "^1.9", features = ["hex"] }
```

```suggestion
serde_with = { version = "1.9", features = ["hex"] }
```

Don't use `^` in Cargo.tomls, it is implied by default, and spelling it explicitly is non-idiomatic.